### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by Maxx1234** [640-KB/GLaBIOS#2]
Any chance for the following features:

-Expansion ROM search end segment at FE00h?
-RTC detection at 2C0h?

BTW, impressive work!

Best regards

**Comment by 640-KB** [comment]
Yes, option ROMs up to FE00 already done and will be checked in with the next update.

I had actually planned for RTC clock (INT 1Ah) support initially but it isn't going to fit in the 8K space. So I am considering them as option ROMs instead, hopefully in the not too distant future.

**Comment by 640-KB** [comment]
**Update:**

This functionality will be included in the upcoming [GLaTICK Companion ROM](https://github.com/640-KB/GLaTICK).

**Comment by 640-KB** [comment]
**Update 2:**

Pre-release testing versions are available on the GLaTICK [pre-release page](https://github.com/640-KB/GLaTICK/releases).

**Comment by 640-KB** [comment]
Closing this issue since the feature is now available in the [**GLaTICK Companion ROM**](https://github.com/640-KB/GLaTICK/).
